test/fast/accountTree.test.ts

```diff
@@ -8,7 +8,7 @@ import { ZERO_BYTES32 } from "../../ts/constants";
 
 let DEPTH: number;
 let BATCH_DEPTH: number;
-describe("Account Tree", async () => {
+describe.only("Account Tree", async () => {
```

Comment:
@ChihChengLiang This ideally would fail in CI from linting, could use https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-exclusive-tests.md although I don't think we have eslint setup. Can we add eslint rules to run through prettier?

Reply:
Adding the no-exclusive eslint rules sounds like a good idea! Forgetting to remove "only" is one of the biggest nightmares for me. Feel free to create an issue for this.

Reply:
https://github.com/thehubbleproject/hubble-contracts/issues/554